src/main/java/com/actiontech/dble/backend/datasource/PhysicalDBPool.java

```diff
@@ -343,6 +343,11 @@ public void doHeartbeat() {
         for (PhysicalDatasource source : all) {
             if (source != null) {
                 source.doHeartbeat();
+                for (PhysicalDatasource[] physicalDatasource : source.getDbPool().standbyReadSourcesMap.values()) {
```

Comment:
standbyReadSourcesMap is a property of this PhysicalDBPool, start the standbyReadSources heartbeat out of the loop to avoid uselessly call